python/private/py_info.bzl

```diff
@@ -140,6 +142,31 @@ A depset of import path strings to be added to the `PYTHONPATH` of executable
 Python targets. These are accumulated from the transitive `deps`.
 The order of the depset is not guaranteed and may be changed in the future. It
 is recommended to use `default` order (the default).
+""",
+        "site_packages_symlinks": """
```

Comment:
We should add a note that this is still unstable and may be removed/changed in between minor version releases of `rules_python`.